.github/actions/docker-build/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: docker-build
+description: Reusable action to build repos' docker images
+
+runs:
+  using: composite
+  steps:
+  - name: cd/build-docker
+    run: make build-image
+    shell: bash
+
+  - name: cd/scan-docker-security
+    uses: aquasecurity/trivy-action@9ab158e8597f3b310480b9a69402b419bc03dbd5 # v0.8.0
+    with:
+      image-ref: "mattermost/backup-restore-tool"
+      format: "table"
+      ignore-unfixed: true
+      vuln-type: "os,library"
+      severity: "CRITICAL"
+
+  - name: cd/docker-push
+    run: ./scripts/push-docker.sh
```

Comment:
Would suggest making this part of the Actual Pipeline, and refrain from calling external bash scripts. 
Even better reusing an existing action to build a docker image.